.github/workflows/release.yml

```diff
@@ -30,14 +30,13 @@ jobs:
         id: daily-version
       - if: steps.daily-version.outputs.created
         name: Create release
-        uses: actions/github-script@v6
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            await github.request(`POST /repos/${{ github.repository }}/releases`, {
-              tag_name: "${{ steps.daily-version.outputs.version }}",
-              generate_release_notes: true
-            });
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd distribution
+          zip -r refined-github.zip *
```

Comment:
@yakov116 I wonder if we should rename it to clarify that this isn't the official way to install it. Like "refined-github-local-install.zip"

We should probably also append the official download links to the releases as well, like:

> Download: [16px chrome icon], [16px firefox icon], [16px safari icon]